lib/fog/openstack/volume_v1.rb

```diff
@@ -43,11 +47,14 @@ class V1 < Fog::Volume::OpenStack
         request :delete_volume_type
         request :get_volume_type_details
 
-        request :create_volume_snapshot
```

Comment:
In this case the request files named "create_volume_snapshot.rb" should be deleted. "create_snapshot.rb" is used instead, which matches the pattern of the existing delete_snapshot.rb, update_snapshot.rb and list_snapshots.rb files.